docs/book/src/developer/providers/implementers-guide/generate_crds.md

```diff
@@ -85,7 +85,7 @@ make manifests
 
 The cluster API CRDs should be further customized:
 
-- [Apply the contract version label to support conversions](../contracts.md#api-version-labels)
+- [Apply the contract version label to support conversions](../contracts.md#api-version-labels) (required to apply _any_ custom resource of your provider)
```

Comment:
```suggestion
- [Apply the contract version label to support conversions](../contracts.md#api-version-labels) (required to apply to _any_ custom resource of your provider)
```

Reply:
I meant that the labels are required before one can deploy the CRs. Reworded accordingly. The linked doc page already ensures that all CRs have the labels (using kustomize `commonLabels: {...}`)